opentelemetry-otlp/src/exporter/tonic/logs.rs

```diff
@@ -86,8 +84,12 @@ impl LogExporter for TonicLogsClient {
         Ok(())
     }
 
-    fn shutdown(&mut self) -> OTelSdkResult {
-        match self.inner.take() {
+    fn shutdown(&self) -> OTelSdkResult {
+        let handle = tokio::runtime::Handle::try_current()
```

Comment:
this is not so clean. We already require users to create the provider from inside a tokio runtime. Maybe we can extend that guideline for shutdown also?

https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/examples/basic-otlp/README.md